src/main/java/com/puppycrawl/tools/checkstyle/StatelessModule.java

```diff
@@ -0,0 +1,10 @@
+package com.puppycrawl.tools.checkstyle;
+
+/**
+ * This interface means that the module does not contain mutable state and
+ * may be safely used from the separate threads at the same time.
```

Comment:
You go into detail on the others, but this one only gets one sentence.
Can you expand on `mutable state`?